index.js

```diff
@@ -0,0 +1,16 @@
+const mobileNav = document.getElementById('mobile_nav');
+const mobileMenuOpen = document.getElementById('mobile_menu_open');
+const mobileNavClose = document.getElementById('mobile_menu_close');
+
+function ShowMobileNav() {
+  mobileNav.classList.remove('hidden');
+  mobileMenuOpen.classList.add('hidden');
+}
+
+function HideMobileNav() {
```

Comment:
Kindly make the necessary changes so that the menu hides after you the user chooses one of the three options 🤓 The website does display the right section but the menu does not disappear.